src/Stripe.net/Services/FileLinks/StripeFileLinkService.cs

```diff
@@ -0,0 +1,79 @@
+namespace Stripe
+{
+    using System.Collections.Generic;
+    using System.Net;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Stripe.Infrastructure;
+
+    public class StripeFileLinkService : StripeService
+    {
+        private static string classUrl = Urls.BaseUrl + "/file_links";
+
+        public StripeFileLinkService()
+            : base(null)
+        {
+        }
+
+        public StripeFileLinkService(string apiKey)
+            : base(apiKey)
+        {
+        }
+
+        public virtual StripeFileLink Create(StripeFileLinkCreateOptions createOptions, StripeRequestOptions requestOptions = null)
+        {
+            return CreateAsync(createOptions, requestOptions).Result;
```

Comment:
I've used a style I'd like to see us adopt for new services (and apply to the existing services):
1. Methods are in alphabetical order (like in tests)
2. Non-async methods simply call their async version + `.Result`, as suggested by @jaymedavis in #940.

Reply:
Are we sure it works across all .NET frameworks?

Also, if we approve this, should we convert the whole library once and for all?

Reply:
Thanks @ob-stripe for keeping things nice and neat and caring about the details. Warms my ❤️

Reply:
@remi-stripe The library almost [does this anyway](https://github.com/stripe/stripe-dotnet/blob/master/src/Stripe.net/Infrastructure/Requestor.cs#L63) on non async calls. Notice it uses the addition of `GetAwaiter().GetResult()`. I believe this was determined good practice from #1145. [This stack overflow post](https://stackoverflow.com/questions/17284517/is-task-result-the-same-as-getawaiter-getresult) sheds some light about the differences... which makes me wonder if [this change](https://github.com/stripe/stripe-dotnet/pull/1145/files#diff-f4f707e39cfd8d37180bd288044fc813R64) was necessary.